index.bs

```diff
@@ -2089,6 +2091,48 @@ To <dfn>await a navigation</dfn> given |context|, |request|, |wait condition|, a
 
 </div>
 
+#### The browsingContext.Locator Type #### {#type-browsingContext-Locator}
+
+[=remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+browsingContext.Locator = (
```

Comment:
Should we have `JavaScriptLocator` in here?

Reply:
@AutomatedTester I would like to have it in the spec, but for this go-round, I can remove it. Or someone else can from the branch, I'm not that wedded to it.

Reply:
Removed in subsequent commit.